com.ca.lsp.cobol/lsp-core-domain/pom.xml

```diff
@@ -75,33 +75,8 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
-
     <build>
         <plugins>
-            <plugin>
```

Comment:
Why did you delete this configuration? Does it present somewhere else?

Reply:
The problem here was that we don't need to produce a jar artifact for the domain module so I remove it.
Plus the main class was in another package.
I will add also Filip for the review

Reply:
Agreed, this was unnecessary, just duplicating what is already present in lsp-service-cobol